src/library_gl.js

```diff
@@ -583,6 +597,7 @@ var LibraryGL = {
     },
 
     deleteContext: function(contextHandle) {
+      if (!contextHandle) return;
```

Comment:
Good question: I cannot remember anymore - it may have been to give graceful `free(0);`-like semantics, or it may have been something else. I'll remove this line, and see if it yells on something. Hopefully it won't be needed, and we can just save a few bytes.